### ISSUE TITLE
Randomly hang ScrollIntoView() function

### ISSUE THREAD

**Issue by jaekook-neonesia** [go-rod/rod#265]
**Rod Version:** v.0.76.6
## Steps to reproduce
- What did you do?
 Naviate URL then try to input text

- What have you tried to solve the problem?
Trace(true) on browser option

## Expected behavior

What do you expect to get?(Success case)
[rod] 2020/11/04 09:12:31 [eval] {"js":"rod.element","params":["input[name='username']"]}
[rod] 2020/11/04 09:12:31 [eval] {"js":"rod.element","params":["input[name='username']"]}
[rod] 2020/11/04 09:12:31 [input] "scroll into view"
[rod] 2020/11/04 09:12:33 [input] "input AAAAAAAAAAAAAAAA"
[rod] 2020/11/04 09:12:35 [eval] {"js":"rod.element","params":["input[name='password']"]}
[rod] 2020/11/04 09:12:35 [input] "scroll into view"
[rod] 2020/11/04 09:12:37 [input] "input BBBBBBBBBBBBB"

## Actual result(Failed case :  Just hang. Cant scroll into view.)
[rod] 2020/11/04 09:48:48 [eval] {"js":"rod.element","params":["input[name='username']"]}
[rod] 2020/11/04 09:48:48 [eval] {"js":"rod.element","params":["input[name='username']"]}
[rod] 2020/11/04 09:48:48 [eval] {"js":"rod.element","params":["input[name='username']"]}

What did you actually get?
Hang,  if there wasn't "show scroll into view"

Randomly happen.

**Comment by ysmood** [comment]
Can you provide minimal code for us to reproduce your issue?

**Comment by jaekook-neonesia** [comment]
`
	l := launcher.MustNewRemote(envChromeURL).Headless(true)

	browser := rod.New().Client(l.Client()).Slowmotion(2 * time.Second).Trace(true).MustConnect()
	defer browser.MustClose()

	page := browser.MustPage("")
	page.MustNavigate("https://vpn.klikbca.com/+CSCOE+/logon.html")
	elm, err := page.Element("input[name='username']")
	if err != nil {
		log.Warn("VPN Input not available.")
		return nil
	}

	elm.MustWaitStable()

	page.MustElement("input[name='username']").MustInput("vpnID")  // this line random hang with ScrollIntoView()
	page.MustElement("input[name='password']").MustInput("vpnPW")
	page.MustElement("tbody > tr > td > a > img").MustClick()

	page.MustWaitNavigation()
	page.MustElement("#menu-1").MustClick()
        ........
`

**Comment by ysmood** [comment]
Thank you for your report! It's not a bug. The reason why is because the `MustInput` will wait for the element to appear before the actual input. But the page has two same inputs, and you are waiting for the one that will never be visible 😂 

![image](https://user-images.githubusercontent.com/1415488/98195276-a9269200-1f64-11eb-897c-f4fbcfd2176b.png)

Solution is simple, just select the correct one:

```go
	page := rod.New().MustConnect().MustPage("")

	page.MustNavigate("https://vpn.klikbca.com/+CSCOE+/logon.html")
	elm, err := page.Element("#wrapper #username")
	if err != nil {
		log.Fatal("VPN Input not available.")
	}

	elm.MustWaitStable()

	page.MustElement("#wrapper #username").MustInput("vpnID") // this line random hang with ScrollIntoView()
	page.MustElement("#wrapper #password_input").MustInput("vpnPW")
	page.MustElement("#wrapper tbody > tr > td > a > img").MustClick()

	page.MustWaitNavigation()
	page.MustElement("#menu-1").MustClick()
```

**Comment by ysmood** [comment]
But I will release a new version with a better trace log, so you will aware of this kind of auto-waiting

**Comment by jaekook-neonesia** [comment]
Thank you so much for your fast answer.

**Comment by ysmood** [comment]
@jaekook-neonesia released `v0.78.2`. Now you will see the log like:

```txt
[rod] 2020/11/05 13:59:14 [eval] {"js":"rod.element","params":["#username"]}
[rod] 2020/11/05 13:59:14 [eval] {"js":"rod.element","params":["#username"]}
[rod] 2020/11/05 13:59:14 [eval] {"js":"rod.element","params":["#username"]}
[rod] 2020/11/05 13:59:15 [eval] {"js":"rod.element","params":["#username"]}
[rod] 2020/11/05 13:59:17 [eval] {"js":"rod.element","params":["#username"]}
[rod] 2020/11/05 13:59:17 [eval] {"js":"rod.visible","this":"input#username.form-control.textbox-bca"}
[rod] 2020/11/05 13:59:17 [eval] {"js":"rod.visible","this":"input#username.form-control.textbox-bca"}
[rod] 2020/11/05 13:59:18 [eval] {"js":"rod.visible","this":"input#username.form-control.textbox-bca"}
```

The `[rod] 2020/11/05 13:59:17 [eval] {"js":"rod.visible","this":"input#username.form-control.textbox-bca"}` is the log for wait-visible.
